resources/conf/config.toml.template

```diff
@@ -455,6 +455,8 @@ enabled = true
   retryInterval = 5
   # Skip SSL verification
   skipSSLVerification = false
+  # Send revision update success message
+  sendRevisionUpdate = false
```

Comment:
What do you think about `sendRevisionAck`. The word `Update` can lead to an misunderstanding that we are not accepting anymore revision updates from the control plane.

Reply:
Yes, improved the description to address the above concern. This is already merged to the choreo branch.

Reply:
This applies to the method names too. Don't you think `SendRevisionUpdateAck` and `SendRevisionUndeployAck` would be less confusing? Since the main branch has diverged a lot considering most other improvements.. what do you think?

Reply:
fixed